updates SeePopNotServed tool for GP service

andybell opened the pull request:
- fixes syntax errors (indentation + missing quote)
- adds time module to the import 
- arcpy.env.overwriteOutput set to true to allow for existing layers to be overwritten by new outputs
- changes loc from a hard path to the parent directory using os abspath
- SeePopNotServed output saved in_memory (swap to hard path in gp service)

andybell committed:
replaces tabs with spaces to fix IndentationErrors & adds mssing closing quote

andybell committed:
env overwrite set to true, SeePopNotServed output to in_memory, working directory set from os to be parent folder

dcgull commented:
Could you please add an explanation of changes made to the server config as well? I think this is the best place to document that.

andybell committed:
removed population grid parameter from SeePopNotServed, hardcoded raster output path to publish gp service

andybell committed:
rm commented out lines for the old Param2

andybell commented:
Few additional things to get the tool running as a GP service:
- Removed population raster parameter from SeePopNotServed to simplify user experience
- Changed raster output from a in_memory dataset to a hard coded path in order to view the result in a map service
- Registered folder as a data store on the server
- Publishing the gp tool as a service did not copy over the `Data\ToolData.gdb` geodatabase. Manually copied the geodatabase to `C:\arcgisserver\directories\arcgissystem\arcgisinput\SeePopNotServed.GPServer\extracted\v101\wpdx-toolset-master\`
